infra/scripts/release-patch.sh

```diff
@@ -0,0 +1,269 @@
+    --unreleased-only  \
+    --no-issues  \
+    --bug-labels kind/bug  \
+    --enhancement-labels kind/feature  \
+    --breaking-labels compat/breaking  \
+    -t $GH_TOKEN \
+    --max-issues 1 -o \
+    | sed -n '/## \[v'"$MAJOR.$MINOR.$PATCH"'\]/,$p' \
+    | sed '$d' | sed '$d' | sed '$d' | tr -d '\r' >> temp \
+    && sed '1d' ../../CHANGELOG.md >> temp && mv temp ../../CHANGELOG.md
+    git diff ../../CHANGELOG.md
+    echo "Check CHANGELOG.md carefully and fix any errors. In particular, make sure the new enhancements/PRs/bugfixes aren't already listed somewhere lower down in the file."
+    read -p "Once you're done checking, continue to commit the changelog (y) or exit (n)? " choice
+    case "$choice" in
+        y|Y ) commit_changelog ;;
+        n|N ) exit ;;
+        * ) exit ;;
+    esac ;
+}
+echo "Step 2: Updating CHANGELOG.md"
+if grep -q "https://github.com/feast-dev/feast/tree/v$MAJOR.$MINOR.$PATCH" ../../CHANGELOG.md ; then
+    read -p "CHANGELOG.md appears updated. Skip this step (y/n)? " choice
+    case "$choice" in
+        y|Y ) echo "Skipping this step" ;;
+        n|N ) update_changelog ;;
+        * ) update_changelog ;;
+    esac ;
+else
+    update_changelog ;
+fi
+
+tag_commit()
+{
+    echo "Tagging commit"
+    git tag v$MAJOR.$MINOR.$PATCH
+    echo "Commit tagged"
+}
+echo "Step 3: Tag commit"
+if git tag | grep -q "v$MAJOR.$MINOR.$PATCH" ; then
+    read -p "The tag already exists. Skip this step (y/n)? " choice
+    case "$choice" in
+        y|Y ) echo "Skipping this step" ;;
+        n|N ) tag_commit ;;
+        * ) tag_commit ;;
+    esac ;
+else
+    tag_commit ;
+fi
+
+echo "Step 4: Push commits and tags"
+push_commits()
+{
+    echo "Pushing commits"
+    git push origin v$MAJOR.$MINOR-branch
+    echo "Commits pushed"
+}
+echo "Step 4a: Push commits"
+if git status | grep -q "nothing to commit" ; then
+    echo "The commits appear pushed. Skipping this sub-step"
+else
+    read -p "Commits are not pushed. Continue (y) or skip this sub-step (n)? " choice
+    case "$choice" in
+        y|Y ) push_commits ;;
+        n|N ) echo "Skipping this sub-step" ;;
+        * ) echo "Skipping this sub-step" ;;
+    esac ;
+fi
+
+push_tag()
+{
+    echo "Pushing tag"
+    git push origin v$MAJOR.$MINOR.$PATCH
+    echo "Tag pushed"
+}
+echo "Step 4b: Push tag"
+if git ls-remote --tags origin | grep -q "v$MAJOR.$MINOR.$PATCH" ; then
+    read -p "The tag appears pushed. Skip this sub-step (y/n)? " choice
+    case "$choice" in
+        y|Y ) echo "Skipping this sub-step" ;;
+        n|N ) push_tag ;;
+        * ) push_tag ;;
+    esac ;
+else
+    read -p "The tag is not pushed. Continue (y) or skip this sub-step (n)? " choice
+    case "$choice" in
+        y|Y ) push_tag ;;
+        n|N ) echo "Skipping this sub-step" ;;
+        * ) echo "Skipping this sub-step" ;;
+    esac ;
+fi
+
+read -p "Now wait for the CI to pass. Continue (y) or exit and fix the problem (n)? " choice
```

Comment:
Ran into 
```
Now wait for the CI to pass. Continue (y) or exit and fix the problem (n)? y
./release-patch.sh: line 193: Moving on to the next step: command not found
```

Reply:
You shouldn't be getting this after bunch of changes below. Closing this comment for now.